keyboards/xd60_rev3/keymaps/default/keymap.c

```diff
@@ -0,0 +1,46 @@
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
```

Comment:
This isn't actually needed, and is included by default already. 

```suggestion
```

Reply:
Sure!